README.md

```diff
@@ -1,135 +1,106 @@
-the [Google Developer Console][google-developer-drive] (click the
-"Enable" button near the top). Then you'll need to create a Google
-OAuth client ID. From
-the
-[Google Developer Credentials Console][google-developer-credentials],
-create a new OAuth Client ID. If you're developing locally (via
-localhost), choose "Other"; otherwise, choose "Web application". Once
-you have the client ID, store it in the Meteor shell by running:
-
-```js
-> ServiceConfiguration.configurations.upsert({service: 'google'}, {
-    $set: {
-      clientId: 'CLIENT ID',
-      secret: 'SECRET',
-      loginStyle: 'popup',
-    },
-  })
-```
-
-Once that's done, manually navigate to `/setup`
-(e.g. http://localhost:3000/setup). From there, you'll be able to link
-an account to the OAuth application, giving Jolly Roger permissions to
-create documents on behalf of that account.
-
-(In production, we have a dedicated Google account that owns both the
-OAuth application and the Drive credentials)
-
-[airbnb-javascript]: https://github.com/airbnb/javascript
-[babeljs]: http://babeljs.io
-[collection2]: https://atmospherejs.com/aldeed/collection2
-[es6]: https://github.com/lukehoban/es6features
-[google-developer-credentials]: https://console.developers.google.com/apis/credentials
-[google-developer-drive]: https://console.developers.google.com/apis/api/drive.googleapis.com/overview
-[ESLint]: http://eslint.org/
-[meteor install]: https://www.meteor.com/install
-[React]: https://facebook.github.io/react/
-[react-router]: https://github.com/rackt/react-router
-[roles]: https://atmospherejs.com/nicolaslopezj/roles
-[simple-schema-chaining]: https://github.com/aldeed/meteor-simple-schema#combining-simpleschemas
+Jolly Roger is a coordination tool for collaboratively solving puzzlehunts like the [MIT Mystery Hunt][http://web.mit.edu/puzzle/www/].
+The team Death and Mayhem created Jolly Roger in 2015 for the 2016 MIT
+Mystery Hunt, and it is still actively maintained and in production service.
+
+At its core, it allows tracking puzzles and guesses in an ever-evolving hunt
+structure, and provides a chat room and a Google spreadsheet for each
+puzzle.
+
+
+## Features
+
+### Automated spreadsheet creation/sharing
+
+Like many Hunt collaboration tools, Jolly Roger automatically creates a
+Google Sheet for each puzzle entered into a hunt.  If a user has linked
+their Google account to Jolly Roger, Jolly Roger will additionally share the
+sheet with that user on first load, which will cause the user's cursor to
+show their name, rather than "Anonymous Animal".
+
+Users who do not link a Google account (or don't have or use one) can still
+participate in the sheet anonymously.
+
+### Per-puzzle chat rooms
+
+Jolly Roger provides its own persistent chat with each puzzle.  No need to
+constantly jump between the spreadsheet and a third-party chat service.  No
+losing relevant discussion to 10000-message retention limits in the middle
+of the Hunt.  Supports Markdown formatting.
```

Comment:
Technically it's a variant of Slack's fake-Markdown (`_` and `*` are not italic and bold respectively in true markdown). That's going to end up on the list of things we'll have to deal with on Discord, which uses true Markdown.

Reply:
Oh, that's a really good point; I'd missed that before.